src/main/webapp/assets/style.css

```diff
@@ -0,0 +1,94 @@
+:root {
+  --dark-green: #667d51;
+  --light-teal: #abcbbc;
+  --light-green: #b9c9a7;
+  --light-maroon: #a98585;
+  --dark-maroon: #955757;
+  --peach: #f2ddc9;
+  --auburn: #ce853f;
+  --tan: #e4c492;
+  --off-white: #f2f0f0;
+}
+
+html,
+body {
+  background-color: var(--off-white);
+  font-family: 
+    'Roboto', 
+    sans-serif;
+  color: #000000;
+  font-size: 1.2rem;
+  margin: 0;
+  padding: 0;
+  min-width: 300px;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  height: 5rem;
+}
+
+.logo {
+  letter-spacing: 1px;
+  font-size: larger;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-around;
+  padding: 0px;
+}
+
+.navlink {
+  list-style: none;
+  margin: 1rem;
+}
+
+.navlink a {
+  text-decoration: none;
+  color: #000000;
+}
+
+.navlink a:hover {
+  color: var(--dark-green);
+}
+
+.burger {
+  font-size: 1.2em;
+  display: none;
+}
+
+@media screen and (max-width: 678px) {
+  .burger {
+    display: block;
+  }
+
+  .nav {
```

Comment:
Nit: should be no indentation here

Reply:
There should be because it's style under a specific media screen!

Reply:
oh my gosh you're right, sorry about that!